plugins/content_distribution/providers/tvinci/lib/admin/TvinciProfileConfiguration.php

```diff
@@ -102,5 +110,52 @@ protected function layoutForm($order)
 					'order' => $order++,
 				)
 		);
+
+		$this->addElement('select', 'schema_id', array(
+			'label'			=> 'Schema ID:',
+			'multioptions' => array(
+				'' => '2',
+				'1' => '1',
```

Comment:
Is it possible to give a meaningful name to each value?
Also, why empty string considered as 2, set 2 as the default if that's what you meant.